tests/foreman/ui_airgun/test_provisioningtemplate.py

```diff
@@ -68,8 +68,5 @@ def test_positive_clone(session):
         assert session.provisioningtemplate.search(
             clone_name)[0]['Name'] == clone_name
         template = session.provisioningtemplate.read(clone_name)
-        cloned_template_os_list = [
-            el.split()[0] for el
```

Comment:
What was the root cause of failure? Theoretically, if we use os name + split os title on ui and pick first element - they should be equal. Your approach is better anyways, i'm just curious

Reply:
in the UI the OS is listed as title not name, and we can use that directly without any other manipulations.

Reply:
seems this has been changed, since the test was written.

Reply:
So the root cause wasn't here, but in `session.provisioningtemplate.clone` command - clone screen contained full os name with version but we were searching for name only. Change from os name to os title resolved the issue.